app/src/main/kotlin/com/wire/android/model/DialogState.kt

```diff
@@ -0,0 +1,6 @@
+package com.wire.android.model
+
+sealed class DialogState<out T : Any> {
```

Comment:
What do you think of adding some docs here? With the intended purpose of this class :D